src/main/resources/view/appView.fxml

```diff
@@ -4,7 +4,7 @@
 <?import javafx.scene.control.TitledPane?>
 <?import javafx.scene.layout.BorderPane?>
 
-<BorderPane fx:id="root" style="-fx-background-color: #f4fffb" minHeight="600.0" minWidth="800.0"
+<BorderPane fx:id="root" style="-fx-background-color: #f4fffb" minHeight="900.0" minWidth="1200.0"
```

Comment:
I'd reduce height. App is too big on my screen and it bugs when i try to resize it.

Reply:
Reduced. Please let me know if now is ok

Reply:
It's okay now.